### PR TITLE
Use 'archived' field to only try to archive unarchived things

### PR DESCRIPTION
**Pull request by jbothma**

### CODE REVIEW

File: gazettes/archive.py [hunk 1]
```diff
@@ -188,6 +188,9 @@ def archive(args):
                 archive_put(cached_gazette_path, archived_gazette.archive_path)
             if not args.cache_gazettes:
                 os.remove(cached_gazette_path)
+            webscraped_sesh.expunge(webgazette)
```
Comment (longhotsummer) [inline gazettes/archive.py]: Why is this needed?

Comment (jbothma) [inline gazettes/archive.py]: It's bound to the session I read it from but I want to update it in the
session where I create the archived gazette instance. Expunge detaches it
from the original session so that I can add it to the archive session. An
alternative is session.merge, which will try to read it up from the DB as
well, which I think is superfluous, or manually reading it up by PK, which
I think is also unnecessary.

Comment (longhotsummer) [inline gazettes/archive.py]: Ah, got it. Why separate sessions? So we can support separate dbs?

Comment (jbothma) [inline gazettes/archive.py]: Nope - so an error later in processing doesn't fail to commit everything
done in one big session. Not that big a deal most of the time when they're
just trickling in but I expected lots of unexpected errors with all the
variance in input data.
